apps/transmitmedia.cpp

```diff
@@ -156,6 +156,8 @@ class SrtStatsJson : public SrtStatsWriter
         output << "{\"bandwidth\":" << mbpsBandwidth << '}' << endl;
         return output.str();
     }
+
+    virtual ~SrtStatsJson() override {};
```

Comment:
Destructor is not obligatory in the derived polymorphic class if its body would be empty. The "not in charge" destructor would be generated anyway. The problem could be only with no virtual destructor in the root class, which could make that the latter not be called.

Reply:
Resolved.